### DISCUSSION TITLE
Are all the db-scheduler capabilities enabled for SAP HANA database?

### DISCUSSION THREAD

**Discussion by ashwganes** [kagkarlsson/db-scheduler#437] [Q&A] [answered]
Are all the db-scheduler capabilities enabled for SAP HANA database?

**Comment by kagkarlsson** [comment, marked as answer]
`lock-and-fetch` polling strategy is only available for Postgres, so no. Also not familiar with SAP HANA, but db-scheduler works fine for Sql Server.
